### PR TITLE
updating esc faq

### PR DESCRIPTION
**Pull request by arunkumar611**

### CODE REVIEW

File: content/docs/esc/faq.md [hunk 1]
```diff
@@ -13,36 +13,34 @@ aliases:
...
 
-Pulumi ESC will be generally available in 2024.
+In other words,
 
-## What will be the pricing of Pulumi ESC?
+- When using the Pulumi ESC Document Editor, each definition of fn::secret:* and fn::open::* (except Pulumi-stacks provider) is counted as a secret.
+- When using the Pulumi ESC Table Editor, each secret and provider config type (except Pulumi-stacks) is counted as a secret.
 
-During the preview, the product is free to use. When the product is made generally available, it will have a pricing model that is competitive with the market.
+The number of secrets only from the latest environment revision is counted towards your billing.
```
Comment (jdavredbeard) [inline content/docs/esc/faq.md:33]: Would reword as "Only the secrets from the latest environment revision are counted towards your billing."

File: content/docs/esc/faq.md [hunk 2]
```diff
@@ -13,36 +13,34 @@ aliases:
...
+Check out our [public roadmap](https://github.com/orgs/pulumi/projects/44/) page.
 
-We encourage you to create a [feature request](https://github.com/pulumi/esc/issues/new/choose) or upvote [existing open requests](https://github.com/pulumi/esc/issues).
+If you are interested in a particular feature, we encourage you to create a [feature request](https://github.com/pulumi/esc/issues/new/choose) or upvote [existing open requests](https://github.com/pulumi/esc/issues).
 
-## Can a Self-hosted Pulumi customer use this?
+## Can I Self-host Pulumi ESC?
 
-Pulumi ESC is currently not supported for self-hosted customers. [Contact sales](/contact/?form=sales) if you'd like to participate in our self-hosted beta for ESC when it is available.
+Yes. [Contact sales](/contact/?form=sales) us to get a demo or trial self-hosted Pulumi ESC.
```
Comment (jdavredbeard) [inline content/docs/esc/faq.md:43]: is the `us` in this line supposed to be here?

Comment (arunkumar611) [reply content/docs/esc/faq.md:43]: I'll update

File: content/docs/esc/faq.md [hunk 3]
```diff
@@ -13,36 +13,34 @@ aliases:
   - /docs/pulumi-cloud/esc/faq
 ---
 
-## Is it safe to use Pulumi ESC for my production stacks?
+## Why did Pulumi launch ESC?
 
-Yes, Pulumi ESC is served using the same [highly secure](/security/pulumi-cloud-security-whitepaper.pdf) and highly available infrastructure as the rest of Pulumi Cloud. The service is currently in preview as we are rapidly iterating and improving the feature set. It is unlikely that we will make any breaking changes. Should we introduce a new API, we are dedicated to ensuring optimal compatibility to prevent any interruptions to your infrastructure.
+We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks. Pulumi ESC brings the same software engineering approach to secrets and configuration, allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks. just like Pulumi IaC. It can be used for all applications and infrastructure - with or without Pulumi IaC.
```
Comment (thtmnisamnstr) [inline content/docs/esc/faq.md:18]: > We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks.

Add a comma before causing.

> We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets, causing sprawl and duplications across stacks.

-----

> Pulumi ESC brings the same software engineering approach to secrets and configuration, allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks. just like Pulumi IaC.

This is garbled.

**Review by thtmnisamnstr — APPROVED** [review]

File: content/docs/esc/faq.md [hunk 4]
```diff
@@ -13,36 +13,34 @@ aliases:
...
-## Why did Pulumi launch ESC?
+## What is the pricing of Pulumi ESC?
+
+See our [pricing page](https://www.pulumi.com/pricing/) for details.
 
-We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks. Pulumi ESC is a natural extension of Pulumi IaC by allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks.
+## What counts as a secret towards pricing?
 
-## When will Pulumi ESC be generally available?
+Secret includes both [static secrets](/docs/esc/get-started/store-and-retrieve-secrets/), [dynamic login credentials](/docs/esc/integrations/dynamic-login-credentials/) and [dynamic secrets](/docs/esc/integrations/dynamic-secrets/).
```
Comment (cnunciato) [inline content/docs/esc/faq.md:26]:
```suggestion
Only Pulumi ESC-managed secrets count towards pricing. This includes [static secrets](/docs/esc/get-started/store-and-retrieve-secrets/), [dynamic login credentials](/docs/esc/integrations/dynamic-login-credentials/) and [dynamic secrets](/docs/esc/integrations/dynamic-secrets/).
```

File: content/docs/esc/faq.md (hunk 3 above)
Comment (arunkumar611) [reply content/docs/esc/faq.md]: Updated

File: content/docs/esc/faq.md [hunk 5]
```diff
@@ -13,36 +13,34 @@ aliases:
...
 
-Pulumi ESC will be generally available in 2024.
+In other words,
 
-## What will be the pricing of Pulumi ESC?
+- When using the Pulumi ESC Document Editor, each definition of fn::secret:* and fn::open::* (except Pulumi-stacks provider) is counted as a secret.
+- When using the Pulumi ESC Table Editor, each secret and provider config type (except Pulumi-stacks) is counted as a secret.
```
[the remark this answers is not in the document]
Comment (arunkumar611) [reply content/docs/esc/faq.md]: Updated.

File: content/docs/esc/faq.md [hunk 6]
```diff
@@ -13,36 +13,31 @@ aliases:
...
+## What counts as a secret towards pricing?
 
-Pulumi ESC will be generally available in 2024.
+Secrets include [static secrets](/docs/esc/get-started/store-and-retrieve-secrets/), [dynamic login credentials](/docs/esc/integrations/dynamic-login-credentials/) and [dynamic secrets](/docs/esc/integrations/dynamic-secrets/).
 
-## What will be the pricing of Pulumi ESC?
+In other words, when using Pulumi ESC's Document Editor, each definition of fn::secret:: and fn::open::* (except [Pulumi-stacks provider](/docs/esc/integrations/infrastructure/pulumi-iac/pulumi-stacks/)) is counted as a secret.
 
-During the preview, the product is free to use. When the product is made generally available, it will have a pricing model that is competitive with the market.
+Only the secrets from the latest environment revision are counted towards your bill.
```
Comment (cnunciato) [inline content/docs/esc/faq.md:30]: Super clear, thanks!

File: content/docs/esc/faq.md [hunk 7]
```diff
@@ -13,36 +13,31 @@ aliases:
...
+See our [pricing page](https://www.pulumi.com/pricing/) for details.
 
-## When will Pulumi ESC be generally available?
+## What counts as a secret towards pricing?
 
-Pulumi ESC will be generally available in 2024.
+Secrets include [static secrets](/docs/esc/get-started/store-and-retrieve-secrets/), [dynamic login credentials](/docs/esc/integrations/dynamic-login-credentials/) and [dynamic secrets](/docs/esc/integrations/dynamic-secrets/).
 
-## What will be the pricing of Pulumi ESC?
+In other words, when using Pulumi ESC's Document Editor, each definition of fn::secret:: and fn::open::* (except [Pulumi-stacks provider](/docs/esc/integrations/infrastructure/pulumi-iac/pulumi-stacks/)) is counted as a secret.
```
Comment (cnunciato) [inline content/docs/esc/faq.md:28]:
```suggestion
In other words, when using Pulumi ESC's document editor, each definition of `fn::secret` and `fn::open::*` (except with the [pulumi-stacks provider](/docs/esc/integrations/infrastructure/pulumi-iac/pulumi-stacks/)) is counted as a secret.
```

**Review by cnunciato — APPROVED** [review]
Couple little suggestions; leaving an approval so you can merge afterward. Thanks!
